Validation/RecoEgamma/plugins/TkConvValidator.cc

```diff
@@ -142,9 +142,9 @@ TkConvValidator::TkConvValidator( const edm::ParameterSet& pset )
     g4_simTk_Token_  = consumes<edm::SimTrackContainer>(pset.getParameter<edm::InputTag>("simTracks"));
     g4_simVtx_Token_ = consumes<edm::SimVertexContainer>(pset.getParameter<edm::InputTag>("simTracks"));
 
-    tpSelForEff_Token_ = consumes<TrackingParticleCollection> (
+    tpSelForEff_Token_ = consumes<TrackingParticleRefVector> (
```

Comment:
It would be good if somebody from EGM could review this `TkConvValidator` part (but I don't know who to ask).